src/app/(main)/layout.module.css

```diff
@@ -0,0 +1,3 @@
+.offsetForStickyHeader {
```

Comment:
Where is the class applied?

Reply:
The changes was added to a diff PR so I have to adjust. give it two sec 👍